poc/Makefile

```diff
@@ -1,2 +1,29 @@
-all:
+.PHONY: install lint tests all
+
+lint/pyflakes:
```

Comment:
How do you guys run these linting tools?

Reply:
I think most of them are available through `brew`.

Reply:
Not installation part. So we just manually invoke those commands?

Reply:
Yes.